README.md

```diff
@@ -8,54 +8,59 @@ The plugin sets up a custom post type for Charts and launches the Simplechart ap
 
 When the post is rendered on the front end, this same data is used to bootstrap redrawing the same chart.
 
+### Installation on WordPress.com VIP
+
+1. Place the plugin in a directory inside your theme, e.g.  `mytheme/inc/wordpress-simplechart`
+1. Follow the **Local app setup commands** steps below
+1. Add this line to your theme's `functions.php` to "activate" the plugin:
+
+````
+require_once( get_template_directory() . '/inc/wordpress-simplechart/simplechart.php' );
+````
+
+### Installation on non-VIP sites
+
+1. Install and activate [Media Explorer](https://github.com/Automattic/media-explorer)
+1. Install and activate the Simplechart plugin
+1. Follow the **Local app setup commands** steps below
+
 ### Usage
 
-1. Install and activate [Media Explorer](https://github.com/Automattic/media-explorer) if you're not on WordPress.com
-1. Install and activate Simplechart
 1. Your WP Admin area should now have a custom post type for Charts.
 1. Click the "Launch Simplechart App" button to create a new chart.
 1. When you're happy with your new chart, click "Send to WordPress" button
 1. Save the post in WordPress
 1. You can now embed the Chart in any post by selecting it from the Charts section in the Media Manager, which will drop a shortcode into the post content.
 
-### Installation
-
-To secure the postMessage interface between wp-admin and the the Simplechart JS app, there are a few different ways to configure the plugin depending on how your site is hosted.
-
-#### Case #1: `http://*.wordpress.com` (https is cool, too)
-
-Congratulations on being a WordPress.com VIP! `*.wordpress.com` domains are whitelisted.
-
-Place the plugin in a directory inside your theme, e.g.  `mytheme/plugins/wordpress-simplechart`
-
-In your theme's `functions.php`:
-`require_once( __DIR__ . '/plugins/wordpress-simplechart/simplechart.php' );`
-
-The Simplechart JS app will be loaded from the default URL:
-`http://simplechart.io/#/simplechart`
-
-#### Case #2: Any other domain
-
-For non-VIP sites, the JS app and plugin must be on the same origin. You can load the WordPress plugin either from your `wp-content/plugins` directory or by requiring in your theme as above.
+### Local app setup commands
 
-Then clone https://github.com/alleyinteractive/simplechart into a directory in your theme like `mytheme/plugins/wordpress-simplechart/app`
+Do this from the root of the `wordpress-simplechart` plugin directory, either in your theme or in `wp-plugins`:
 
-You might need to switch to the `gh-pages` branch
+````
+$ git clone git@github.com:alleyinteractive/simplechart.git
+$ mv simplechart/client/pages/ app
+$ rm -rf simplechart
```

Comment:
This seems like a good candidate for reducing complexity. Maybe we can jigger either a npm or bower package to provide the `pages` dir, and require it that way?